src/Teams/Controllers/MultipleAnswerQuestionController.cs

```diff
@@ -51,12 +58,16 @@ public IActionResult EditMultipleAnswerQuestion(Guid id)
         [HttpGet]
         public IActionResult ShowMultipleAnswerQuestionForm()
         {
+            _logger.LogInformation("");
             return View("AddMultipleAnswerQuestion");
         }
 
         [HttpPost]
         public IActionResult AddMultipleAnswerQuestion([FromBody] MultipleQuestionAddModel multipleAnswersQuestionDTO)
         {
+            IEnumerable<string> answers = multipleAnswersQuestionDTO.QuestionAnswers.Select(a => a.AnswersText);
+            _logger.LogInformation($"QuestionText: {multipleAnswersQuestionDTO.QuestionText}|Answers: {String.Join(", ", answers)}");
```

Comment:
usually by styles guides we use not String or Int32, we are using string and just int. So please change to string.Join

Reply:
good catch!